samples/nrf9160/http_update/common/src/update.c

```diff
@@ -8,8 +8,6 @@
 #include <drivers/flash.h>
 #include <nrf_modem.h>
 #include <modem/lte_lc.h>
-#include <modem/at_cmd.h>
-#include <modem/at_notif.h>
 #include <modem/nrf_modem_lib.h>
```

Comment:
I think `modem/nrf_modem_lib.h`  and `nrf_modem.h` do not need to be included any longer